_integration-schemas/ms-teams/v1/conversation_posts.md

```diff
@@ -0,0 +1,92 @@
+---
+tap: "ms-teams"
+version: "1"
+key: ""
+
+name: "conversation_posts"
+doc-link: "https://docs.microsoft.com/en-us/graph/api/conversationthread-list-posts?view=graph-rest-beta&tabs=http"
+singer-schema: "https://github.com/singer-io/tap-ms-teams/blob/master/tap_ms_teams/schemas/conversation_posts.json"
+description: |
+  The `{{ table.name }}` table contains information about the posts within a conversation thread in your Microsoft account.
+
+replication-method: "Key-based Incremental"
+
+api-method:
+    name: "List posts"
+    doc-link: "https://docs.microsoft.com/en-us/graph/api/conversationthread-list-posts?view=graph-rest-beta&tabs=http"
```

Comment:
```suggestion
    doc-link: "https://docs.microsoft.com/en-us/graph/api/conversationthread-list-posts?view=graph-rest-1.0&tabs=http"
```

[This table actually uses the V1 API ](https://github.com/singer-io/tap-ms-teams/blob/master/tap_ms_teams/streams.py#L477)